lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -334,7 +334,7 @@ def commit(method, action, parameters)
         )
       rescue ActiveMerchant::ResponseError => e
         response = e.response.body.present? ? parse(e.response.body) : { 'status' => { 'response_code' => e.response.msg } }
-        message = response['status'].slice('message', 'response_code').values.compact_blank.first || ''
+        message = response['status'].slice('message', 'response_code').values.select(&:present?).first || ''
```

Comment:
This one changed from `.compact_blank` to `.select(&:present?)` but other places in to code changed from `.compact_blank` to `.compact`. Why is this one different?

Reply:
Good callout, the main difference between them is here we are compacting an array where in the other places we were compacting an object. I updated the other locations to be this implementation though since it will be more apples to apples with the .compact_blank method.